aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -81,6 +81,11 @@ impl BlockExecutorTransactionOutput for AptosTransactionOutput {
             TransactionStatus::Retry,
         )))
     }
+
+    /// Return the amount of gas consumed by the transaction.
+    fn gas_used(&self) -> u64 {
```

Comment:
super nit / subjective - maybe consumed_gas or charged_gas would be better?

Reply:
I think `gas_used` is the term that is used everywhere, so better to be consistent.

Reply:
cool!